EMOBON00092/manual_example_ro-crate-metadata.json

```diff
@@ -54,174 +41,216 @@
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged_CDS.faa.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged_CDS.ffn.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_1_clean.fastq.trimmed.fasta.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_1_clean.fastq.trimmed.qc_summary.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_2_clean.fastq.trimmed.fasta.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_2_clean.fastq.trimmed.qc_summary.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/RNA-counts.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/fastp.html.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/final.contigs.fa.dvc"},
+                        {"@id": "EMOBON00092/DBH.merged.cmsearch.all.tblout.deoverlapped"},
+                        {"@id": "EMOBON00092/DBH.merged.fasta"},
+                        {"@id": "EMOBON00092/DBH.merged.motus.tsv"},
+                        {"@id": "EMOBON00092/DBH.merged.qc_summary"},
+                        {"@id": "EMOBON00092/DBH.merged.unfiltered_fasta"},
+                        {"@id": "EMOBON00092/DBH.merged_CDS.faa"},
+                        {"@id": "EMOBON00092/DBH.merged_CDS.ffn"},
+                        {"@id": "EMOBON00092/DBH_1_clean.fastq.trimmed.fasta"},
+                        {"@id": "EMOBON00092/DBH_1_clean.fastq.trimmed.qc_summary"},
+                        {"@id": "EMOBON00092/DBH_2_clean.fastq.trimmed.fasta"},
+                        {"@id": "EMOBON00092/DBH_2_clean.fastq.trimmed.qc_summary"},
+                        {"@id": "EMOBON00092/RNA-counts"},
+                        {"@id": "EMOBON00092/fastp.html"},
+                        {"@id": "EMOBON00092/final.contigs.fa"},
                         {"@id": "functional-annotation"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/run.yml.dvc"},
+                        {"@id": "EMOBON00092/run.yml"},
                         {"@id": "sequence-categorisation"},
-                        {"@id": "taxonomy-summary"}
+                        {"@id": "taxonomy-summary"},
+                        {"@id": "EMOBON00092/provenance.json"}
                     ] 
     },
     {
-        "@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.cmsearch.all.tblout.deoverlapped.dvc",
+        "@id": "EMOBON00092/provenance.json",
+        "@type": ["File", "CPMProvenanceFile"],

```

Comment:
feels like the URI for type CPMProvenanceFile should also be added to the extended @context